independent-projects/bootstrap/app-model/src/main/java/io/quarkus/paths/PathVisit.java

```diff
@@ -44,6 +44,16 @@ default URL getUrl() {
         }
     }
 
+    /**
+     * Path relative to the root of the tree as a string with {@code /} as a path element separator.
+     * This method calls {@link #getRelativePath(String)} passing {@code /} as an argument.
+     *
+     * @return path relative to the root of the tree as a string with {@code /} as a path element separator
+     */
+    default String getRelativePath() {
+        return getRelativePath("/");
```

Comment:
@aloubyansky  is this also working when fs uses "\" ?

Reply:
Yes, it's just a string with '/' as a separator.